lib/vrl/stdlib/src/del.rs

```diff
@@ -118,71 +118,3 @@ impl fmt::Display for DelFn {
         f.write_str("")
     }
 }
-
-/*
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::map;
-    use std::str::FromStr;
-
-    #[test]
-    fn del() {
-        let cases = vec![
-            (
-                // String field exists
-                map!["exists": "value"],
-                Ok(value!("value")),
-                DelFn::new(Path::from("exists")),
-            ),
-            (
-                // String field doesn't exist
-                map!["exists": "value"],
-                Ok(value!(null)),
-                DelFn::new(Path::from("does_not_exist")),
-            ),
-            (
-                // Array field exists
-                map!["exists": value!([1, 2, 3])],
-                Ok(value!([1, 2, 3])),
-                DelFn::new(Path::from("exists")),
-            ),
-            (
-                // Null field exists
-                map!["exists": value!(null)],
-                Ok(value!(null)),
-                DelFn::new(Path::from("exists")),
-            ),
-            (
-                // Map field exists
-                map!["exists": map!["foo": "bar"]],
-                Ok(value!(map!["foo": "bar"])),
-                DelFn::new(Path::from("exists")),
-            ),
-            (
-                // Integer field exists
-                map!["exists": 127],
-                Ok(value!(127)),
-                DelFn::new(Path::from("exists")),
-            ),
-            (
-                // Array field exists
-                map!["exists": value!([1, 2, 3])],
-                Ok(value!(2)),
-                DelFn::new(vrl::Path::from_str(".exists[1]").unwrap().into()),
-            ),
-        ];
-
-        let mut state = state::Program::default();
-
-        for (object, exp, func) in cases {
-            let mut object: Value = object.into();
-            let got = func
-                .resolve(&mut ctx)
-                .map_err(|e| format!("{:#}", anyhow::anyhow!(e)));
-
-            assert_eq!(got, exp);
-        }
-    }
-}
-*/
```

Comment:
Should these be added back (I presume we can't use the `test_function` macro, but without the macro it should still be possible right?

Reply:
Good point. I meant to revisit it.